File: go/day48.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
	"regexp"
)

func main() {
	fmt.Println("Please input your email")

	reader := bufio.NewReader(os.Stdin)
	emailAddress, _ := reader.ReadString('\n')
	emailAddress = strings.TrimSpace(emailAddress)

	match, _ := regexp.MatchString("\\@.+\\.",emailAddress)

	fmt.Println("Valid Email:", match);
}

File: cSharp/day48.cs
using System;
using System.Text.RegularExpressions;


Console.WriteLine("Please input email");

string emailAddress = Console.ReadLine();

Console.WriteLine("Valid email: ");

Regex pattern = new Regex("\\@.+\\.");

if (pattern.Match(emailAddress).Success)
{
    Console.WriteLine("True");
}
else
{
    Console.WriteLine("False");
}